tests/test-webapps/test-servlet-spec/test-spec-webapp/src/main/webapp/WEB-INF/jetty-web.xml

```diff
@@ -2,7 +2,10 @@
 <!DOCTYPE Configure PUBLIC "-//Mort Bay Consulting//DTD Configure//EN" "http://www.eclipse.org/jetty/configure_9_3.dtd">
 
 <Configure class="org.eclipse.jetty.webapp.WebAppContext">
+  <Get name="servletContext">
+    <Call name="log"><Arg>The test-spec webapp is deployed. DO NOT USE IN PRODUCTION!</Arg><Arg></Arg></Call>
+  </Get>
   <Get class="org.eclipse.jetty.util.log.Log" name="rootLogger">
-    <Call name="warn"><Arg>test-spec webapp is deployed. DO NOT USE IN PRODUCTION!</Arg></Call>
+    <Call name="info"><Arg>WEB-INF/lib/jetty-util.jar logging used!</Arg></Call>
```

Comment:
Why this additional log line about `jetty-util.jar`? Debug leftover?

Reply:
The first one is the general warning about production.
The second one is a usage of the WEB-INF/lib/jetty-util logging... which should be the old version.... however it is currently not there, so I need to comment out this line like the others with a TODO to put it back when there is a fixed 9.4 release